composer.json

```diff
@@ -2,21 +2,22 @@
     "name": "phpcr/phpcr-shell",
     "description": "Shell for PHPCR",
     "require": {
-        "symfony/console": "~2.3",
+        "symfony/console": "~2.4",
         "jackalope/jackalope": "~1.1",
         "phpcr/phpcr": "~2.1",
         "phpcr/phpcr-utils": "~1.2",
-        "symfony/finder": "~2.3",
-        "symfony/serializer": "~2.3",
-        "symfony/yaml": "~2.3",
-        "symfony/dependency-injection": "~2.3",
+        "symfony/finder": "~2.4",
+        "symfony/serializer": "~2.4",
+        "symfony/yaml": "~2.4",
```

Comment:
why are you dropping the Symfony LTS support for all these components ?

Note that given ``symfony/expression-language`` is fully standalone, it is possible to install the 2.4 component in a symfony 2.3 project to use it (you will not have the integrations in places where Symfony itself uses the expression-language of course)